Updated based on internal feedback (#1536)
astronomer/docs: astro/cross-account-role-setup.md

### Permissions used to manage data plane resources

As the Astro service changes, Astronomer support might change the role policy attached to the cross-account role. Astronomer support informs all organizations by email before implementing any changes. To make sure you receive these notifications, subscribe to the [Status Page](https://status.astronomer.io/). 

The following table lists the permissions required by Astro to manage the cross-account role.

> | Permissions                                           | Purpose                                                                                                                        |

Reviewer: @dnarain-astro Why have we decided to remove specific permission information from this page? Are the uses of each policy self-evident?

Reviewer 2: @jwitz this is the comment from @dnarain-astro in [PR 1541](https://github.com/astronomer/docs/pull/1541):

_Feedback has been that having each permission listed in this document is too AWS-centric (i.e. we are basically regurgitating what each API family does on AWS). Instead, we will point users to the policy in the public S3 bucket and they can examine this on their own._

> When a data plane is activated on AWS, a cross-account IAM role and supporting policies are created. These define the set of actions the control plane is permitted to perform in the management of the data plane.

Reviewer 2: ```suggestion
When you activate a data plane on AWS, a cross-account IAM role and supporting policies are created to define the set of actions the control plane is permitted to perform in the management of the data plane.
```

> Astronomer follows the principles of least privilege access and policies are defined to support the functionality of the entire Astro product. You can preview this role by examining [the CloudFormation template](https://astro-cross-account-role-template.s3.us-east-2.amazonaws.com/customer-account-prerelease.yaml) that will be used to create this role.                                                                          |

Reviewer 2: ```suggestion
Astronomer has implemented least privilege access policies to support Astro functionality. To preview the roles that support these policies, see the [CloudFormation template](https://astro-cross-account-role-template.s3.us-east-2.amazonaws.com/customer-account-prerelease.yaml).                                                                          |
```

> ### Notification of changes to the cross account role

Reviewer 2: ```suggestion
### Notification of changes to the cross-account role
```

Reviewer: Maybe "Receive notification" to keep it action oriented?

> From time-to-time, it may be necessary to make changes to these policies to ensure continued operation and evolution of the Astro platform. 

Reviewer 2: ```suggestion
Occasionally, Astronomer will change its policies to ensure the continued operation and development of Astro. 
```

> Astronomer will notify users with an Organization Owner role at least 14 days prior to any changes to a policy governing the cross-account IAM role that result in an *expansion* of access. The notification will be sent via email, reflect the change(s) to the policy, and include a brief explanation for the change. 

Reviewer 2: ```suggestion
Fourteen days prior to a change to the policy governing the cross-account IAM role that results in an expansion of access, users with an Organization Owner role will receive a notification email from Astronomer support. The notification email describes the policy change an explanation of why it was necessary. 
```

> Astronomer may *reduce* the access available to the cross-account role as part of standard change management approaches without any notifications.

Reviewer 2: ```suggestion
Astronomer can reduce the access available to the cross-account role without notification.
```
